TRANSLATING.md

```diff
@@ -16,6 +16,12 @@ Most of our internationalization decisions and discussions happen on Discord. Jo
 
 Translations all live in this GitHub repository. You can add and update them by creating a pull request or reviewing pending translations in the "Pull Requests" tab. Read on to find out more!
 
+#### Minimum quality standards
+
+We are only able to maintain multiple translations thanks to our amazing team of volunteers. Therefore, we may close PRs that don't meet our minimum quality standards, as we don't have enough bandwidth to handle them. 
+
+Besides expecting translators to be Astro users, all translations should be close to a native speaker's grammar level and well adapted, without big changes to the meaning and structure of a page. This ensures reviewers will only have to make small (if any) suggestions to a translation before merging, making our current workflow sustainable.
```

Comment:
```suggestion
We are only able to maintain multiple translations of the docs thanks to our amazing team of volunteers, who are prepared to spot typos and fix the occasional grammatical error. Translations requiring significant editing cannot be accepted and therefore, we may close PRs that don't meet our minimum quality standards.

We do expect translators to be Astro users, so that they are familiar with Astro terms and concepts. Translations should be faithful representations of the English text, at a native speaker's grammar level, without changes to the meaning and structure of a page. Changes to content are only made in the English documentation, and translations that attempt to "improve" the documentation will not be accepted. (If you believe there is a problem with the English documentation, then please create a GitHub Issue so it can be addressed in the English docs.) This ensures translation reviewers will only have to make small (if any) suggestions to a translation before merging, making our current workflow sustainable.
```